src/highspy/__init__.py

```diff
@@ -1,89 +1,78 @@
-from ._core import (
-    ObjSense,
-    MatrixFormat,
-    HessianFormat,
-    SolutionStatus,
-    BasisValidity,
-    HighsModelStatus,
-    HighsPresolveStatus,
-    HighsBasisStatus,
-    HighsVarType,
-    HighsOptionType,
-    HighsInfoType,
-    HighsStatus,
-    HighsLogType,
-    IisStrategy,
-    IisBoundStatus,
-    HighsSparseMatrix,
-    HighsLp,
-    HighsHessian,
-    HighsModel,
-    HighsInfo,
-    HighsOptions,
-    _Highs,  # type: ignore
-    HighsIis,
-    HighsSolution,
-    HighsObjectiveSolution,
-    HighsBasis,
-    HighsRangingRecord,
-    HighsRanging,
-    kHighsInf,
-    kHighsIInf,
-    HIGHS_VERSION_MAJOR,
-    HIGHS_VERSION_MINOR,
-    HIGHS_VERSION_PATCH,
-    simplex_constants,  # type: ignore
-    cb,  # type: ignore
-    kSolutionStatusNone,
-    kSolutionStatusInfeasible,
-    kSolutionStatusFeasible,
-    kBasisValidityInvalid,
-    kBasisValidityValid,
-)
+from highspy._core import \
+    HighsObjSense, \
+    HighsMatrixFormat, \
+    HighsHessianFormat, \
+    HighsSolutionStatus, \
+    HighsBasisValidity, \
+    HighsModelStatus, \
+    HighsPresolveStatus, \
+    HighsBasisStatus, \
+    HighsVarType, \
+    HighsOptionType, \
+    HighsInfoType, \
+    HighsStatus, \
+    HighsLogType, \
+    HighsIisStrategy, \
+    HighsIisBoundStatus, \
+    HighsSparseMatrix, \
+    HighsLp, \
+    HighsHessian, \
+    HighsModel, \
+    HighsInfo, \
+    HighsOptions, \
+    _Highs, \
+    HighsIis, \
+    HighsSolution, \
+    HighsObjectiveSolution, \
+    HighsBasis, \
+    HighsRangingRecord, \
+    HighsRanging, \
+    kHighsInf, \
+    kHighsIInf, \
+    HIGHS_VERSION_MAJOR, \
+    HIGHS_VERSION_MINOR, \
+    HIGHS_VERSION_PATCH, \
+    simplex_constants, \
+    cb
 
 from .highs import Highs
 
-__all__ = [
-    "__doc__",
-    "ObjSense",
-    "MatrixFormat",
-    "HessianFormat",
-    "SolutionStatus",
-    "BasisValidity",
-    "HighsModelStatus",
-    "HighsPresolveStatus",
-    "HighsBasisStatus",
-    "HighsVarType",
-    "HighsOptionType",
-    "HighsInfoType",
-    "HighsStatus",
-    "HighsLogType",
-    "IisStrategy",
-    "IisBoundStatus",
-    "HighsSparseMatrix",
-    "HighsLp",
-    "HighsHessian",
-    "HighsModel",
-    "HighsInfo",
-    "HighsOptions",
-    "_Highs",
-    "Highs",
-    "HighsIis",
-    "HighsSolution",
-    "HighsObjectiveSolution",
-    "HighsBasis",
-    "HighsRangingRecord",
-    "HighsRanging",
-    "kHighsInf",
-    "kHighsIInf",
-    "HIGHS_VERSION_MAJOR",
-    "HIGHS_VERSION_MINOR",
-    "HIGHS_VERSION_PATCH",
-    "simplex_constants",
-    "cb",
-    "kSolutionStatusNone",
-    "kSolutionStatusInfeasible",
-    "kSolutionStatusFeasible",
-    "kBasisValidityInvalid",
-    "kBasisValidityValid",
-]
+__all__ = ["__doc__",
```

Comment:
Is there actually a reason to import all these symbols as soon as a user imports `highspy`?